pydis_site/templates/events/pages/code-jams/9/frameworks.html

```diff
@@ -16,39 +16,53 @@
+            <a href="https://fastapi.tiangolo.com/advanced/websockets" class="card-footer-item"><i class="fas fa-book"></i>&ensp;Documentation</a>
+            <a href="https://github.com/tiangolo/fastapi" class="card-footer-item"><i class="fab fa-github"></i>&ensp;GitHub</a>
+        </div>
+    </div>
+
+    <div class="card mb-4">
+        <div class="card-content">
+            <div class="content">
+                <p class="subtitle">Starlette</p>
+                <p>Starlette is a lightweight ASGI framework/toolkit, which is ideal for building async web services in Python.
                 </p>
             </div>
         </div>
         <div class="card-footer">
-            <a href="https://websockets.readthedocs.io/en/stable" class="card-footer-item"><i class="fas fa-book"></i>&ensp;Documentation</a>
-            <a href="https://github.com/aaugustin/websockets" class="card-footer-item"><i class="fab fa-github"></i>&ensp;GitHub</a>
+            <a href="https://www.starlette.io/websockets" class="card-footer-item"><i class="fas fa-book"></i>&ensp;Documentation</a>
+            <a href="https://github.com/encode/starlette" class="card-footer-item"><i class="fab fa-github"></i>&ensp;GitHub</a>
         </div>
     </div>
+
     <div class="card mb-4">
         <div class="card-content">
             <div class="content">
-                <p class="subtitle">Flask-SocketIO</p>
-                <p class="is-italic">Flask-SocketIO gives Flask applications access to low latency bi-directional communications between the clients and the server.
+                <p class="subtitle">websockets</p>
+                <p>websockets is a library for building both WebSocket clients and servers with focus on simplicity and performance.
                 </p>
             </div>
         </div>
         <div class="card-footer">
-            <a href="https://flask-socketio.readthedocs.io/en/latest" class="card-footer-item"><i class="fas fa-book"></i>&ensp;Documentation</a>
-            <a href="https://github.com/miguelgrinberg/flask-socketio" class="card-footer-item"><i class="fab fa-github"></i>&ensp;GitHub</a>
+            <a href="https://websockets.readthedocs.io/en/stable" class="card-footer-item"><i class="fas fa-book"></i>&ensp;Documentation</a>
+            <a href="https://github.com/aaugustin/websockets" class="card-footer-item"><i class="fab fa-github"></i>&ensp;GitHub</a>
         </div>
     </div>
+
     <div class="card mb-4">
         <div class="card-content">
             <div class="content">
                 <p class="subtitle">Django Channels</p>
-                <p class="is-italic">Channels is a project that takes Django and extends its abilities beyond HTTP - to handle WebSockets, chat protocols, IoT protocols, and more.
-                    It’s built on a Python specification called ASGI.
+                <>Django Channels adds WebSocket-support to Django - built on ASGI like other web frameworks.
```

Comment:
```suggestion
                Django Channels adds WebSocket-support to Django - built on ASGI like other web frameworks.
```

Reply:
This is supposed to be a paragraph tag that is supposed to be closed below. It appears that I accidentally also removed the `p`.